starlette_admin/templates/forms/tinymce.html

```diff
@@ -0,0 +1,28 @@
+<div class="{%if error%}{{field.error_class}}{%endif%}">
+    <textarea name="{{field.id}}" id="tinymce-editor" data-bs-toggle="autosize"
+        class="form-control {%if error%}{{field.error_class}}{%endif%}" {{ field.input_params()
+        }}>{%if data%}{{data}}{%endif%}</textarea>
+    {% if field.help_text %}
+    <small class="form-hint">{{field.help_text}}</small>
+    {%endif%}
+</div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/tinymce/6.4.0/tinymce.min.js"
+    integrity="sha512-nv2Ftve23IDZqQhji5P2w17Ch88OR37z6tV6djv8U6hcjpRjDXRypN6sXkN6UQo8S+/qf67LVh1a3COduJIG3Q=="
+    crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+<script>
```

Comment:
Remove from the template and override this -> https://jowilf.github.io/starlette-admin/api/fields/#starlette_admin.fields.BaseField.additional_js_links

Reply:
I will change that.